concordium-consensus/src/Concordium/KonsensusV1/TreeState/LowLevel/LMDB.hs

```diff
@@ -357,13 +308,19 @@ makeDatabaseHandlers treeStateDir readOnly initSize = do
                     [MDB_CREATE | not readOnly]
         return DatabaseHandlers{..}
 
+-- | Default start environment size.
+--  Tree state database sizes for historical protocol versions have been between 7-60 times
+--  the 'defaultEnvSize'.
+defaultEnvSize :: Int
+defaultEnvSize = 2 ^ (27 :: Int) -- 128MB
```

Comment:
We have multiple of these constants, some 64, some 128MB. Why?

Reply:
mistake, there should only be the 128mb one. (that is how it's been before)